### DISCUSSION TITLE
Change history for better traceability in audits

### DISCUSSION THREAD

**Discussion by K8Sudo** [monarc-project/MonarcAppFO#363] [Ideas]
It would be very helpful for traceability in audits if there were a change history showing which user made which changes. This could be added as another entry in the menu bar (dashboard, knowledge base, interview table, ...). Here already some simple list would be enough. Are there any plans to introduce such a feature here.

**Comment by ruslanbaidan** [comment]
This was planned some time ago, delayed, but of course on the table as it is an important feature to be done.
https://github.com/monarc-project/MonarcAppFO/issues/17
The first steps were done to set in the "updater" filed email of a user. This will help to get a user, who did the last modification.
But this is not the history of modifications, which can be achieved with use a separate actions log table (for critical/sensitive data modifications).

**Reply by K8Sudo** [reply]
Thanks for the info. I had not seen the issue yet.
It would be helpful if the overview also provided a simple overview of the risks not yet reviewed/adjusted in the risk management cycle (one year). This issue came up recently with a customer who was audited by KPMG. Here it was questioned whether a review of the assessments of all 5000 risks had actually been carried out. If MONARC as a tool does not offer any support here, this would have to be tracked at great expense using Excel tables.

**Reply by ruslanbaidan** [reply]
The case, you described is a bit different than what Critical Actions Logging task includes.
To implement a view with list of unmodified risks more than 1 year, could be a simple filter used to filter by updated_at field older then 1 year.

**Reply by K8Sudo** [reply]
Sure. But would also be very helpful in the context of audits

**Reply by ruslanbaidan** [reply]
Ok, I've linked it to the ticket
